apps/judicial-system/web/src/routes/Court/RestrictionCase/Confirmation/Confirmation.strings.ts

```diff
@@ -0,0 +1,48 @@
+import { defineMessages } from 'react-intl'
+
+export const confirmation = defineMessages({
+  title: {
+    id: 'judicial.system.court:confirmation.title',
```

Comment:
No it is the same here. Maybe rename the namespace to `core` and `confirmation` to `restriction_case_confirmation`
```suggestion
    id: 'judicial.system.core:restriction_case_confirmation.title',
```